src/LanguageServer/Impl/Resources.resx

```diff
@@ -117,6 +117,12 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="AnalysisProgress_MultipleItemsRemaining" xml:space="preserve">
```

Comment:
If those strings go directly to the client, how do we pick up the culture?

Reply:
It is to be determined.... Unfortunately, LSP does not specify locale, so we'll have to add some custom notification. Which reminds me to file https://github.com/Microsoft/python-language-server/issues/98

Reply:
Can't we have those strings on VSC side and provide only numbers?

Reply:
VSC is not the only client. LSP is public and supported by Eclipse, Vim, Emacs and more. So there is actually a plan to try LS with one of those.

Reply:
Ok, but isn't this part of the LS protocol is python specific and we will have to add something to the client-side anyway?

Reply:
Something like `python/setLocale`. If client doesn't send it, we'll just use English. Also, clients typically expect string in exceptions which they display in UI. We throw with descriptive message when we cannot rename, for example.